com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs

```diff
@@ -7,7 +7,7 @@ namespace UnityEngine.Rendering
     /// <summary>
     /// An Asset which holds a set of settings to use with a <see cref="Volume"/>.
     /// </summary>
-    [HDRPHelpURLAttribute("Volume-Profile")]
+    [CoreRPHelpURL("Volume-Profile", "com.unity.render-pipelines.high-definition")]
```

Comment:
should we move the doc to Core package instead?

Reply:
The docs team should do a pass on this :) @Vic-Cooper , @oleks-k, there are pages from hdrp that are referenced from core. Let us know when you modify them and we will remove this string.